docs/snippets/azure/AppHost/Program.cs

```diff
@@ -5,7 +5,7 @@
 builder.AddAzureAppConfiguration("config");
 builder.AddAzureApplicationInsights("app-insights");
 builder.AddAzureCosmosDB("cosmos");
-builder.AddAzureEventHubs("event-hubs");
+builder.AddAzureEventHubs("event-hubs").AddHub("messages");
```

Comment:
I updated this so the bicep would emit a `message` hub, which matches the code above the bicep snippet in the docs.